projects/components/src/summary-values/summary-values.component.ts

```diff
@@ -0,0 +1,56 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { TypedSimpleChanges } from '@hypertrace/common';
+import { SummaryValueDisplayStyle } from '../summary-value/summary-value.component';
+
+@Component({
+  selector: 'ht-summary-values',
+  styleUrls: ['./summary-values.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="ht-summary-values" data-sensitive-pii>
+      <ht-summary-value
+        [icon]="this.icon"
+        [value]="this.displayValue"
+        [label]="this.label"
+        [tooltip]="this.tooltip"
+        summaryValueDisplayStyle="${SummaryValueDisplayStyle.Text}"
+        class="summary-value"
+      ></ht-summary-value>
+      <ng-container *ngIf="this.additionalValues && this.additionalValues!.length > 0">
+        <div class="additional-values" [htTooltip]="detailsTemplate">
+          <span class="count">+{{ this.additionalValues!.length }}</span>
+        </div>
+        <ng-template #detailsTemplate>
+          <div class="tooltip-contents">
+            <span *ngFor="let value of this.additionalValues" class="value">
```

Comment:
So the tooltip is all the values in sibling spans? Should these be comma separated or anything? If this looks like how I'm imagining, I'd think one tooltip should apply across the whole component listing all values together, rather than separating into two different tooltips.

Reply:
Check the screenshot :)